yktrace/src/tir.rs

```diff
@@ -117,6 +117,10 @@ impl TirTrace {
                 }
             };
 
+            // Initialise VarRenamer's accumulator (and thus also set the first offset) to the
+            // traces most outer number of locals.
+            rnm.init_acc(body.num_locals);
```

Comment:
would this be better as an argument to `VarRename::new()`? It's only done once?

Reply:
The problem is when we create `VarRename` we don't have access to `body` yet (at least not without jumping through some hoops).

Reply:
I see. Never mind then.